Cargo.toml

```
@@ -20,9 +20,9 @@ path = "src/lib.rs"
 [[bin]]
 name = "dofigen"
 path = "src/bin/main.rs"
+required-features = ["bin"]
```

The feature `bin` didn't give enough meaning to final user,
I think naming it `cli` can help people to know the feature it to add the command line features.

```suggestion
required-features = ["cli"]
```